waku.md

```diff
@@ -15,7 +15,6 @@
     - [ABNF specification](#abnf-specification)
     - [Packet Codes](#packet-codes)
     - [Packet usage](#packet-usage)
```

Comment:
where's the ref to data field in parent spec?

also, how do we want to deal with versions here? e.g. in waku.md we write 'using v0.1' of spec etc

Reply:
> where's the ref to data field in parent spec?

wdym?

Reply:
Currently we have `### Whisper Envelope data field (Optional)` in waku spec. There should be a reference from waku.md spec ('parent' spec) that you can look at this other specification for getting X.

Reply:
Ping above